dialogflow/cloud-client/src/main/java/com/example/dialogflow/DetectIntentKnowledge.java

```diff
@@ -104,48 +101,15 @@ public static void detectIntentKnowledge(
           System.out.format(" - Answer: '%s'\n", answer.getAnswer());
           System.out.format(" - Confidence: '%s'\n", answer.getMatchConfidence());
         }
+
+        KnowledgeAnswers answers = sessionsClient
+            .detectIntent(detectIntentRequest)
+            .getQueryResult()
```

Comment:
Why send the same RPC again instead of reusing the previously returned response?

Reply:
Just a miss, good catch!